src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -79,6 +85,20 @@ def load_arguments(self, _):
         configured_default=None
     )
 
+    snapshot_name_arg_type = CLIArgumentType(
+        options_list=['--snapshot-name', '-s'],
+        type=str,
+        help='Name of the App Configuration snapshot.',
+        configured_default=None
+    )
+
+    snapshot_filter_arg_type = CLIArgumentType(
+        options_list=['--filters'],
+        validator=validate_snapshot_filters,
+        nargs='+',
+        help='Space-separated key and label filters used to build an app configuration snapshot. These values should be escaped JSON objects.'
```

Comment:
I wasn't sure what the filters should look like just by reading this message. It's a complicated input format that users may need to figure out by looking at usage samples anyway. 
However, I think this may be slightly clearer:
"Space-separated list of escaped JSON objects that represent the key and label filters used to build an App Configuration snapshot."
What do you think?

Reply:
I believe your suggestion explains it more clearly, thanks! This has been updated accordingly.